tests/hilti/types/regexp/fold-prevention.hlt

```diff
@@ -0,0 +1,13 @@
+# @TEST-EXEC: hiltic -c %INPUT | grep -q 'f(.*::r)'
```

Comment:
Just for clarity, could you change the name of the arg to `f` to something more verbose (e.g., `my_regex`), and match a longer string here, e.g., `grep -q 'f(.*::Test::my_regex)`?